src/openepd/model/specs/base.py

```diff
@@ -50,3 +53,8 @@ def __init__(self, **data: Any) -> None:
     _version_major_match_validator = pyd.validator("ext_version", allow_reuse=True, check_fields=False)(
         validate_version_compatibility("_EXT_VERSION")
     )
+
+
+def set_openepd_unit_validator(validator: QuantityValidator):
```

Comment:
maybe it would be better one single universal setup function like this?

```python
def setup_external_validators(unit_validator: QuantityValidator | None = None, another_validator: Val2 | None = None, ...) -> None:
   ...
```

Reply:
done